src/ui/components/Card/Card.scss

```diff
@@ -41,7 +41,7 @@ $footer-padding: 10px;
 
   border-bottom-left-radius: $border-radius-default;
   border-bottom-right-radius: $border-radius-default;
-  font-variant: all-small-caps;
+  
```

Comment:
This empty line can just be removed, by the way. That way you won't get the linting test failure.

Reply:
Yeah, I know that but Im unable to run docker for testing the changes.

Reply:
You don’t need docker to run this app locally, just node 6: https://github.com/mozilla/addons-frontend

Once you install node you can develop against the dev site using`yarn amo:dev`.